Automattic/Gravatar-SDK-Android
# Updated and regenerated REST API

Closes #135 

### Description

**Note:** This is a huge PR. I'm so sorry about that. It's complicated to split it.

Updated model classes and code with the new REST API definition.

### Testing Steps

- Install the demo app and verify you can load the profile cards for different emails and everything looks as expected (Exactly as it was working before this PR).

https://github.com/Automattic/Gravatar-SDK-Android/assets/6974554/306c9dcb-5d30-445d-82d0-dad9faa95c42
